src/GraphQL.Conventions/Adapters/Engine/GraphQLExecutor.cs

```diff
@@ -133,9 +136,9 @@ public IGraphQLExecutor<ExecutionResult> DisableProfiling()
             return this.EnableProfiling(false);
         }
 
-        public async Task<ExecutionResult> Execute() =>
+        public async Task<ExecutionResult> ExecuteAsync() =>
```

Comment:
Seeing this now, I realise we can skip the `async`, `await` and `ConfigureAwait()`. :-) Simply:
```suggestion
        public Task<ExecutionResult> ExecuteAsync() => _engine.ExecuteAsync(...);
```